src/sagemaker_training/mpi.py

```diff
@@ -379,7 +381,11 @@ def _can_connect(host, port=22):  # type: (str, int) -> bool
     except Exception as e:  # pylint: disable=broad-except
         logger.info("Cannot connect to host %s", host)
 
-        logger.info("Connection failed with exception: \n %s", str(e))
+        logger.info(
+            "Connection failed with exception: \n %s. \
+             Can be ignored for worker when master completes and exits.",
```

Comment:
same comment on inclusive terminology here. please let's check it for all logs

Reply:
we will revisit this in another PR where the improvements to the toolkit is planned.